tests/zfs-tests/tests/functional/cli_root/zfs_clone/zfs_clone_010_pos.ksh

```diff
@@ -206,28 +202,23 @@ for ds in $datasets; do
 done
 names=$(zfs list -rt all -o name,clones $TESTPOOL)
 log_must verify_clones 3 1 $TESTCLONE
-i=1
 for ds in $datasets; do
 	log_must zfs promote $ds
-	((i=i+1))
 done
 log_must local_cleanup
 
 log_note "verify clone list truncated correctly"
-typeset -i j=200
-i=1
 fs=$TESTPOOL/$TESTFS1
+xs=$(python -c 'print("x" * 200)')
 if is_linux; then
 	ZFS_MAXPROPLEN=4096
 else
 	ZFS_MAXPROPLEN=1024
 fi
 log_must zfs create $fs
 log_must zfs snapshot $fs@snap
-while((i <= $(( ZFS_MAXPROPLEN/200+1 )))); do
-	log_must zfs clone $fs@snap $fs/$TESTCLONE$(python -c 'print "x" * 200').$i
-	((i=i+1))
-	((j=j+200))
+for (( i = 1; i <= (ZFS_MAXPROPLEN / 200 + 1); i++ )); do
+	log_must zfs clone $fs@snap $fs/$TESTCLONE$xs.$i
```

Comment:
i'd like update line: `log_must zfs clone $fs@snap $fs/$TESTCLONE$xs.$i` to:
`log_must zfs clone ${fs}@snap $fs/${TESTCLONE}${xs}.$i`